esm/XHR.d.ts

```diff
@@ -1,9 +1,89 @@
-interface GetOptions {
+type XMLHttpRequestBodyInit = string | Blob | BufferSource | FormData | URLSearchParams;
```

Comment:
他所では暗黙的にXHRと表記しているのでこちらも統一して
```suggestion
type XHRBodyInit = string | Blob | BufferSource | FormData | URLSearchParams;
```
としても良いかと思いましたがいかがでしょうか。

Reply:
こちら、xhr.send() 引数の `globalThis.XMLHttpRequestBodyInit` 型を指しています。
`XMLHttpRequestBodyInit` が存在しないというコンパイルエラーが出たため https://github.com/IntimateMerger/im-core/pull/5/commits/dd90d7c7184618ef33eb93a060d61c6693915c6c 時点で手動でエイリアスを作成しました、その後解消した模様です。削除し対応します。ご指摘ありがとうございます。